bytes/tests/integration/test_meta_repository.py

```diff
@@ -170,7 +170,35 @@ def test_filter_raw_on_organization(meta_repository: SQLMetaDataRepository) -> N
     assert len(meta_repository.get_raw(query_filter)) == 1
 
     query_filter.organization = "test2"
-    assert len(meta_repository.get_raw(query_filter)) == 1
+    assert len(meta_repository.get_raw(query_filter)) == 0
+
+
+def test_filter_raw_not_on_organization(meta_repository: SQLMetaDataRepository) -> None:
```

Comment:
Nice tests 👍 It might be even better to test the contents alongside the result counts. While checking lengths is good, it doesn't guarantee that the correct slice (in the right order) is selected. Adding content-based tests ensures a more robust validation of the functionality